RELEASE_NOTES.md

```diff
@@ -2,7 +2,9 @@
 
 ## in develop
 
-* ..
+* WDL: Stages (frags/blocks) reuse applets as well. Previously only tasks were reused. **Breaking**: can break the logic
```

Comment:
How about we change it to: Fragments and blocks reuse applets as well, i.e. they are not rebuilt if the code corresponding to them hasn't been updated in the WDL source file. Previously only tasks..

Reply:
fixed in [98647f0](https://github.com/dnanexus/dxCompiler/pull/291/commits/98647f0cb1881747656c96096189f220c7a0b1d6)